libraries/src/main/kotlin/com/avito/android/ExternalLibrariesExtension.kt

```diff
@@ -32,7 +32,7 @@ abstract class ExternalLibrariesExtension @Inject constructor(private val provid
      * We use exact version to provide consistent environment and avoid build cache issues
      * (AGP tasks has artifacts from build tools)
      */
-    val buildToolsVersion = "29.0.3"
+    val buildToolsVersion = "30.0.3"
```

Comment:
Do we need to update the builder image accordingly?

Reply:
https://github.com/avito-tech/avito-android/blob/94959be2a2641b12bb603f8c1e125765193bbe9c/ci/docker/android-builder/packages.txt#L1-L2

It already supports both